lib/stripe/radar/review.ex

```diff
@@ -10,20 +10,28 @@ defmodule Stripe.Review do
   @type t :: %__MODULE__{
           id: Stripe.id(),
           object: String.t(),
+          billing_zip: String.t(),
           charge: Stripe.id() | Stripe.Charge.t(),
+          closed_reason: String.t(),
           created: Stripe.timestamp(),
           livemode: boolean,
           open: boolean,
+          opened_reason: String.t(),
+          payment_intent: String.t(),
```

Comment:
Would you mind adding `session` (hash), `ip_address` (string), `ip_address_location` (hash) here as well?